src/nodes.coffee

```diff
@@ -270,7 +270,9 @@ exports.Base = class Base
   # as JSON. This is what the `ast` option in the Node API returns.
   # We try to follow the [Babel AST spec](https://github.com/babel/babel/blob/master/packages/babel-parser/ast/spec.md)
   # as closely as possible, for improved interoperability with other tools.
-  ast: (o) ->
+  ast: (o, level) ->
```

Comment:
Similarly to how we just added support for `o.scope`, we now need to support passing `level` on `o`

So allow passing it as a second argument to `ast()` (like the signature of `compileToFragments()`)